duckduckgo/macos-browser
# NetP: Geoswitching Pixels

Task/Issue URL: https://app.asana.com/0/414235014887631/1206107031155633/f

**Description**:

**Steps to test this PR**:
1. None really; just check it compiles (which CI will do). The only changed type is not used from macOS.
---
###### Internal references:
[Pull Request Review Checklist](https://app.asana.com/0/1202500774821704/1203764234894239/f)
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
[Pull Request Documentation](https://app.asana.com/0/1202500774821704/1204012835277482/f)
